### PR TITLE
Display hardcoded

### PR DESCRIPTION
**Pull request by ziniblesssed**

### CODE REVIEW

**Review by CharlesJ10 — APPROVED** [review]

File: capstone/pom.xml [hunk 1]
```diff
@@ -116,7 +116,7 @@
                 <failOnMissingWebXml>false</failOnMissingWebXml>
                 <webResources>
                     <resource>
-                        <directory>${basedir}/target/angular/data-demo/dist/data-demo</directory>
+                        <directory>${basedir}/target/angular/data_demo/dist/data-demo</directory>
```
Comment (ananci) [inline capstone/pom.xml:119]: I thought we had already fixed all of this. Is this still the case @HEAD?

File: capstone/src/main/webapp/script.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    for (var i = 0; i < list.length; i++) {
+      contentListElement.appendChild(
+      createListElement(('Res: '+ list[i].resFileSize +'bytes')));
+      contentListElement.appendChild(
+      createListElement(('Dex: '+ list[i].dexFileSize +'bytes')));
+      contentListElement.appendChild(
+      createListElement(('Lib: '+ list[i].libraryFileSize +'bytes')));
+      contentListElement.appendChild(
+      createListElement(('Ass: '+ list[i].assetsFileSize +'bytes')));
```
Comment (ananci) [inline capstone/src/main/webapp/script.js]: Let's call these something clearer. Like "Assets", "Java Code" (for DEX), etc...

File: capstone/src/main/webapp/script.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+function drawChart() {
+    fetch("/display").then(response => response.json()).then((list) => {
+
+        for ( var i = 0; i < list.length; i++) {
+            var data = google.visualization.arrayToDataTable([
+            ['Content', 'Size'],
+            ['Res',  list[i].resFileSize ],
+            ['Dex',  list[i].dexFileSize],
+            ['Libraries', list[i].libraryFileSize],
+            ['assets', list[i].assetsFileSize],
```
Comment (ananci) [inline capstone/src/main/webapp/script.js]: Make sure these are consistently capitalized / etc and consistent with what you created before.

**Review by ananci — CHANGES_REQUESTED** [review]

File: capstone/pom.xml (hunk 1 above)
Comment (ziniblesssed) [reply capstone/pom.xml:119]: I keep getting errors after the pom.xml conflict we had the other time. This fixed it.

Comment (ananci) [reply capstone/pom.xml:119]: sgtm.

File: capstone/src/main/webapp/script.js (hunk 3 above)
Comment (ziniblesssed) [reply capstone/src/main/webapp/script.js]: Made the change. Skipped my eyes for a second

File: capstone/src/main/webapp/script.js (hunk 2 above)
Comment (ziniblesssed) [reply capstone/src/main/webapp/script.js]: Done. Thanks

File: capstone/src/main/webapp/script.js [hunk 4]
```diff
@@ -11,13 +11,13 @@ function getDisplay() {
       contentListElement.appendChild(
       createListElement(('Dex: '+ list[i].dexFileSize +'bytes')));
       contentListElement.appendChild(
-      createListElement(('Lib: '+ list[i].libraryFileSize +'bytes')));
+      createListElement(('Library: '+ list[i].libraryFileSize +'bytes')));
```
Comment (ananci) [inline capstone/src/main/webapp/script.js]: "Libraries" to match with below.

**Review by HallM — APPROVED** [review]

**Review by ananci — APPROVED** [review]
Nice Work!

**Review by Chronos34 — APPROVED** [review]

**Pull request merged**